src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -86,31 +86,47 @@ private boolean validateCreationDate(LocalDate creationDate, Errors errors, Stri
     }
 
     private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
-                                   TrustDataDto trustDataDto,
-                                   Errors errors,
-                                   String loggingContext) {
+                                    TrustDataDto trustDataDto,
+                                    Errors errors,
+                                    String loggingContext) {
         final String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        final LocalDate trustCreationDate = trustDataDto.getCreationDate();
         final LocalDate trustCeasedDate = trustDataDto.getCeasedDate();
 
-        if (isTrustStillRequired(overseasEntitySubmissionDto, trustDataDto.getTrustId())) {
-            // Cease date of the trust must be 'null' as there are still Individual and/or Corporate BOs associated with this trust
-            if (Objects.nonNull(trustCeasedDate)) {
-                final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
-                setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
-                ApiLogger.infoContext(loggingContext, errorMessage);
+        if (areBeneficialOwnersStillLinkedToThisTrust(overseasEntitySubmissionDto, trustDataDto.getTrustId())) {
+
+            if (isTrustStillInvolvedInOverseasEntityFlagNotNull(trustDataDto.isTrustStillInvolvedInOverseasEntity(), errors, loggingContext)) {
```

Comment:
Testing this branch with the web changes and this is causing an error on the Registration journey as `trust_still_involved_in_overseas_entity` is null. This makes sense and I think the null value is correct, but... API should only be validating the ceased date if on the Update or Remove journey (I think).

Reply:
This issue is now addressed by [this](https://github.com/companieshouse/overseas-entities-api/pull/398) PR which has been merged in.